doc/source/tutorials/bipartite_matching/assets/bipartite_matching.py

```diff
@@ -0,0 +1,42 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+# Assign nodes 0-4 to one side, and the nodes 5-8 to the other side
+g = ig.Graph.Bipartite(
+    [0, 0, 0, 0, 0, 1, 1, 1, 1],
+    [(0, 5), (1, 6), (1, 7), (2, 5), (2, 8), (3, 6), (4, 5), (4, 6)]
+)
+assert(g.is_bipartite())
+
+matching = g.maximum_bipartite_matching()
+
+# Print pairings for each node on one side
+matching_size = 0
+print("Matching is:")
+for i in range(5):
+    print(f"{i} - {matching.match_of(i)}")
+    if matching.match_of(i):
```

Comment:
This happens to work in the current case because `matching.match_of()` never returns 0. However, if you were running this on `for i in range(5, 10)`, the size of the matching would be off by one because `matching.match_of(5)` is 0, and 0 is falsy in Python.

it is safer to use `matching.match_of(i) is not None` or to call `matching.match_of()` with a `Vertex` object (i.e. `g.vs[i]`, in which case `matching.match_of()` returns a `Vertex` object, which is never falsy.